### PR TITLE
SoftPoly update

### PR DESCRIPTION
**Pull request by drfrag666**
Bugfix for invulnerability effect and software gamma by @Erick194.

### CODE REVIEW

File: src/common/rendering/polyrenderer/backend/poly_framebuffer.cpp [hunk 1]
```diff
@@ -177,8 +184,13 @@ void PolyFrameBuffer::Update()
 		if (dst)
 		{
 #if 1
+			//auto copyqueue = std::make_shared<DrawerCommandQueue>(&mFrameMemory);
+			//copyqueue->Push<MemcpyCommand>(dst, pitch / pixelsize, src, w, h, w, pixelsize);
+			//DrawerThreads::Execute(copyqueue);
```
Comment (alexey-lysiuk) [inline src/common/rendering/polyrenderer/backend/poly_framebuffer.cpp:189]: Please remove old commented code.

File: src/common/rendering/polyrenderer/backend/poly_framebuffer.cpp [hunk 2]
```diff
@@ -54,6 +55,8 @@ extern bool gpuStatActive;
 extern bool keepGpuStatActive;
 extern FString gpuStatOutput;
 
+extern gamestate_t wipegamestate;
```
Comment (alexey-lysiuk) [inline src/common/rendering/polyrenderer/backend/poly_framebuffer.cpp:58]: Why was this added? I don't see any usage of it.

**Review by alexey-lysiuk — CHANGES_REQUESTED** [review]
Changes to `src/d_main.cpp` are redundant and unrelated to gamma correction.

**Comment by Erick194** [comment]
totally agree, things that happen when making changes 😅

**Comment by drfrag666** [comment]
Yeah i missed that.

**Comment by drfrag666** [comment]
About those changes in d_main.cpp he made some changes there and then reverted them, they must have been made automatically by the VS editor.  Are they really a problem?

**Comment by alexey-lysiuk** [comment]
Yes, such meaningless changes pollutes results of diff and blame commands greatly. There are more then enough examples of them in the history with 19+k commits. Is it really a problem to revert one changed file? It's a trivial thing to do with git interactive rebase.

**Comment by drfrag666** [comment]
In the end he made the changes himself, i'll restore your tabs later.

**Comment by drfrag666** [comment]
Done editing that commit with tgit.

**Review by alexey-lysiuk — COMMENTED** [review]
@dpjudas should take a look at this.

**Review by alexey-lysiuk — APPROVED** [review]
